.gitignore

```diff
@@ -3,7 +3,6 @@ media/service_image
 media/portfolio_image
 media/doc_images
 static/
-# ldap_config.py
```

Comment:
It should stay here to prevent people to push their own ldap_config.py. You can use `git add -f ldap_config.py` if you really need to change it

Reply:
As now we have env variables, this file should not be ignored anymore. So this line is ok I guess.